src/ui/router/ApplicationRouter.js

```diff
@@ -20,33 +20,36 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-import EventEmitter from 'EventEmitter';
+import { EventEmitter } from 'eventemitter3';
 import LocationBar from 'location-bar';
 import _ from 'lodash';
 
 class ApplicationRouter extends EventEmitter {
   /**
-     * events
-     * change:params -> notify listeners w/ new, old, and changed.
-     * change:path -> notify listeners w/ new, old paths.
-     *
-     * methods:
-     * update(path, params) -> updates path and params at the same time.  Only
-     *   updates specified params, other params are not modified.
-     * updateParams(newParams) -> update only specified params, leaving rest
-     *      intact.  Does not modify path.
-     * updatePath(path);
-
-     * route(path, handler);
-     * start(); Start routing.
-     */
+   * events
+   * change:params -> notify listeners w/ new, old, and changed.
+   * change:path -> notify listeners w/ new, old paths.
+   *
+   * methods:
+   * update(path, params) -> updates path and params at the same time.  Only
+   *   updates specified params, other params are not modified.
+   * updateParams(newParams) -> update only specified params, leaving rest
+   *      intact.  Does not modify path.
+   * updatePath(path);
+   *
+   * route(path, handler);
+   * start(); Start routing.
+   * @param {import('../../../openmct').OpenMCT} openmct
+   */
+
   constructor(openmct) {
     super();
 
     this.locationBar = new LocationBar();
     this.openmct = openmct;
     this.routes = [];
     this.started = false;
+    this.path = undefined;
```

Comment:
can we use null here instead? i think not as downstream we're checking only if it's `undefined`, but i always think it's weird to define something as `undefined`!

Reply:
yes yes yes